views.py

```diff
@@ -100,14 +106,16 @@ def post(self):
     if 'max_groups' in solr_args:
         del solr_args['max_groups']
 
-    solr_args["rows"] = min(int(solr_args.get("rows", [current_app.config.get("VIS_SERVICE_PN_MAX_RECORDS")])[0]), current_app.config.get("VIS_SERVICE_PN_MAX_RECORDS"))
+    data = solr_args["q"]
 
+    solr_args["rows"] = min(int(solr_args.get("rows", [current_app.config.get("VIS_SERVICE_PN_MAX_RECORDS")])[0]), current_app.config.get("VIS_SERVICE_PN_MAX_RECORDS"))
     solr_args['fl'] = ['bibcode,title,first_author,year,citation_count,read_count,reference']
     solr_args['wt'] ='json'
+    solr_args['q'] = '*:*'
 
     headers = {'X-Forwarded-Authorization' : request.headers.get('Authorization')}
 
-    response = client().get(current_app.config.get("VIS_SERVICE_SOLR_PATH") , params = solr_args, headers=headers)
+    response = client().post(current_app.config.get(VIS_SERVICE_BIGQUERY_PATH), params = solr_args, headers=headers, data=data)
```

Comment:
Thanks for finding the earlier (embarrassing) mistakes.
For your two points here: so should I do a multidict that has two fqs, one being bitset?
And I guess I thought bigquery could take any query, but it is limited to a list of bibcodes? So in order to use this I should require that people use the "bibcode" parameter rather than the "solr_query" one, right? (Right now I just assume the params are for a solr query)